### PR TITLE
Update Deno icon

### PR DESCRIPTION
**Pull request by LitoMore**
image

**Origin issue:** #1179

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

FYI: https://github.com/denoland/dotland/pull/1836

### CODE REVIEW

File: icons/deno.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...l="none" fill-rule="evenodd"><path fill="#000" fill-rule="nonzero" d="M256 0a256 256 0 1 0 0 512 256 256 0 1 0 0-512z"/><mask id="b" fill="#fff"><use xlink:href="#a" width="100%" height="100%"/></mask><path d="m71 319 17-63m19.964-94.905 17-63M36.93 221l17-63m72.034 227 17-63m17.408 164.829 17-63m52.628 32.5 17-63M206.257 92.587l17-63m103.138 143.417 17-63m108.787 194.689 17-63M409.124 221l17-63M299.027 54.558l17-63m84.597 94.5 17-63" stroke="#fff" stroke-width="25" fill="#000" fill-rule="nonzero" stroke-linecap="round" mask="url(#b)"/><mask id="d" fill="#fff"><use xlink:href="#c" width="100%" height="100%"/></mask><path d="M252.225 344.418c-86.65 2.61-144.576-34.5-144.576-94.363 0-61.494 60.33-111.145 138.351-111.145 37.683 0 69.532 10.65 94.392 30.092 21.882 17.113 37.521 40.526 45.519 66.312 2.574 8.301 22.863 83.767 61.112 227.295l1.295 4.86-159.793 74.443-1.101-8.063c-8.85-64.778-16.546-113.338-23.076-145.634-3.237-16.004-6.178-27.96-8.79-35.794-1.227-3.682-2.355-6.361-3.303-7.952h-.05z" stroke="#000" stroke-width="12" fill="#fff" fill-rule="nonzero" mask="url(#d)"/><path fill="#000" fill-rule="nonzero" mask="url(#d)" d="M262 187a16 16 0 1 0 0 32 16 16 0 1 0 0-32z"/></g></svg>
```
Comment (dirien) [inline icons/deno.svg:1]: hi @LitoMore,

thanks for the contribution. Can you check the deno.svg please?

```
x attr-1 0:0 Expected attribute 'xmlns' to be "http://www.w3.org/2000/svg", w
               as "https://www.w3.org/2000/svg"
                 At node <svg> (0:0)
  x icon-size Size of <path> must be exactly 24 in one dimension; the size is c
              urrently 460 x 460
  x extraneous Unexpected character(s), most likely extraneous whitespace, dete
               cted in SVG markup
  x icon-centered <path> must be centered at (12, 12); the center is currently 
                  (256, 256)
  x elm Found 0 elements for 'svg > path', expected 1
  x elm 0:91 Element disallowed
               At node <defs> (0:91)
  x elm 0:97 Element disallowed
               At node <path> (0:97)
  x elm 0:164 Element disallowed
                At node <path> (0:164)
  x elm 0:238 Element disallowed
                At node <g> (0:238)
  x elm 0:299 Element disallowed
                At node <path> (0:299)
  x elm 0:390 Element disallowed
                At node <mask> (0:390)
  x elm 0:415 Element disallowed
                At node <use> (0:415)
  x elm 0:471 Element disallowed
                At node <path> (0:471)
  x elm 0:812 Element disallowed
                At node <mask> (0:812)
  x elm 0:837 Element disallowed
                At node <use> (0:837)
  x elm 0:893 Element disallowed
                At node <path> (0:893)
  x elm 0:1358 Element disallowed
                 At node <path> (0:1358)
```

**Review by dirien — CHANGES_REQUESTED** [review]
hi @LitoMore,

left a comment. 

Can you run:

```
npm run svgo -- icons/deno.svg
npm run lint  
```

before re-submitting, so you see instantly any remaining issues.

**Comment by LitoMore** [comment]
@dirien Updated.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -2389,7 +2389,7 @@
         {
             "title": "Deno",
             "hex": "000000",
-            "source": "https://github.com/denoland/deno_website2/blob/892750b99cc260f8b1b2a4bcea59eab9101dd67c/public/logo.svg"
+            "source": "https://github.com/denoland/dotland/blob/717b6815c95b413336bb2bf1ee9177c120eb894c/public/logo.svg"
```
Comment (sachinraja) [inline _data/simple-icons.json:2392]: Great work, this is actually a perfect match! Can you just add `"license": "MIT"` here?

**Review by sachinraja — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 2 above)
Comment (LitoMore) [reply _data/simple-icons.json]: License added. 

> Also does this icon use the changes from your latest commit on denoland/dotland?

Yes. Source link updated.

**Review by sachinraja — APPROVED** [review]

**Pull request merged**